chapters/hub/hub-builder.md

```diff
@@ -0,0 +1,36 @@
+# Jina Hub Builder
+
+Jina’s hub-builder is a simple interface for building & validating Jina Hub executors. It is built on top of jina hub interface. It can be used as a Github action in the CICD workflow, or via CLI.
+
+## Github action usage:
+One can use the `hub builder` as a part of the CI pipeline, to build and test images in the Pull Request. The following YAML could be directly used in the workflow `.github/workflows/hub-builder.yml`.
+```
+name: Hub Builder
+
+on: [pull_request]
+
+jobs:
+  hub-builder:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Jina Hub Image Builder
+        uses: jina-ai/hub-builder@master
+```
+On every new PR, the builder finds modifications in `manifest.yml` recursively and tries to build a Hub image from it, one by one. That means, when you update an image, you must change manifest.yml to trigger the build, e.g. you can simply bump version field in `manifest.yml`.
```

Comment:
- `tries to build a Hub image from it` - what's `it`? (we have two subjects - "file" and "modifications" so need to be more specific here)
- `one by one` - it's building hub images one by one? Or it goes through each modification one by one?
- Wrap manifest.yml in backticks
- e.g. -> i.e.
- the version field